doc/api/fs.markdown

```diff
@@ -350,6 +350,10 @@ created. On POSIX systems, `path` is considered to exist even if it is a symlink
 to a non-existent file. The exclusive flag may or may not work with network file
 systems.
 
+`flags` can also be a number as documented by open(2); commonly used constants
+are available from `require('constants')`.  On Windows, some flags are
+translated to their equivalent ones as accepted by CreateFileW.
```

Comment:
@sindresorhus @XeCycle See my comment above.  Can you comment on whether this line is necessary?